src/start_provision_win/w32_start_provision.c

```diff
@@ -192,6 +192,8 @@ int		do_sync_mom_hookfiles;
 struct batch_request    *saved_takeover_req;
 struct python_interpreter_data  svr_interp_data;
 extern void	*AVL_jctx = NULL;
+long long svr_max_job_sequence_id = 0;
```

Comment:
Why this variable is defined in three other files globally on Windows?

Reply:
Windows build was failing just because of the variable because didn't declared earlier. Can not compile on windows without declaring this variable because some mapping happens between the files.

Reply:
Have you tried including a header file where this variable is declared as `extern`